src/python_framework_v02/troute/nhd_io.py

```diff
@@ -1034,6 +1034,80 @@ def get_channel_restart_from_wrf_hydro(
     return q_initial_states
 
 
+def read_lite_restart(
+    file
+):
+    '''
+    Open lite restart pickle files. Can open either waterbody_restart or channel_restart
+    
+    Arguments
+    -----------
+        file (string): File path to lite restart file
+        
+    Returns
+    ----------
+        df (DataFrame): restart states
+        t0 (datetime): restart datetime
+    '''
+    
+    # open pickle file to pandas DataFrame
+    df = pd.read_pickle(pathlib.Path(file))
+    
+    # extract restart time as datetime object
+    t0 = df['time'].iloc[0].to_pydatetime()
+    
+    return df.drop(columns = 'time') , t0
```

Comment:
New function to quickly read lite restart (pickle) files, returns the a DataFrame of either channel or waterbody states, plus a datetime object of the restart time.